content/format-reduction/index.md

```diff
@@ -0,0 +1,56 @@
+---
+rfc:
+start_date: 2018-07-26
+pr:
+status: draft
+---
+
+# Format reduction
+
+## Summary
+
+This RFC proposes to reduce the amount of representations in Registers.
+
+
+## Motivation
+
+The Registers data model and its REST API is still in motion. The amount of
+formats suggested in the specification and implemented in the reference
+implementation increase unnecessarily the complexity of evolving them. For
+example, we want to evolve who the schema fits into the data model and how it
+is surfaced to the user.
+
+
+## Explanation
+
+The current suggestion in the spec is to implement JSON, YAML, CSV, TSV, HTML
+and Turtle. Although none of them are required, the reference implementation
+has implemented all of them.
+
+There are a few issues to notice:
+
+The Turtle implementation is not quite
+right and the specification is not mature enough to make different
+implementors implement the same thing by following the specification alone.
+
+The YAML representation offers no value on top of the JSON one.
+
+The CSV and TSV are virtually the same.
```

Comment:
> CSV and TSV _implementations_

? (or some similar noun)

Reply:
I would use "representations" for consistency.